src/rules/package-lavamoat-tsup-conforms.ts

```diff
@@ -0,0 +1,34 @@
+import { buildRule } from './build-rule';
+import { RuleName } from './types';
+import { dataConform } from '../rule-helpers';
+
+export default buildRule({
+  name: RuleName.PackageLavamoatTsupConforms,
+  description:
+    'Does the `lavamoat.allowscripts` field in `package.json` conform?',
```

Comment:
I think we ought to be a bit more detailed here. What do you think about:

``` suggestion
    'Does LavaMoat allow scripts for \`tsup>esbuild\`?',

Reply:
It makes sense to me that the description should break the pattern, because this rule works a bit differently than those rules. This rule doesn't just check that the `lavaMoat.allowScripts` field conforms to the module template; if it did that, it wouldn't care what was inside the field. But in this case we are checking something more specific: we want to know whether `tsup>esbuild` is present. So my thought is that the rule description should be more specific to match the extra logic happening here. Does that make sense?